src/cookieConsent/cookieConsent.component.tsx

```diff
@@ -52,20 +51,22 @@ export const CookieConsent = (
       consentCookie &&
       consentCookie.analytics
     ) {
-      ReactGA.initialize(props.analytics.id, {
-        titleCase: false,
-        gaOptions: {
-          cookieExpires: 60 * 60 * 24 * 365, // one year
-          cookieFlags: 'Samesite=None;Secure',
-        },
-      });
-      const page = `${props.location.pathname}${props.location.search}`;
-      ReactGA.set({
-        anonymizeIp: true,
-        page,
-      });
-      // need to send initial pageview
-      ReactGA.pageview(page);
+      //Global Site Tag (gtag.js) - Google Analytics
+      const urlScript = document.createElement('script');
+      urlScript.async = true;
+      urlScript.src =
+        'https://www.googletagmanager.com/gtag/js?id=G-BMV4M8LC8J';
```

Comment:
The `id` query param here should be grabbed from `props.analytics.id` right?

Reply:
Thanks, yeh it should! changes made.